README.md

```diff
@@ -109,7 +109,7 @@ require additional configurations applied to
 
 The following prerequisites are required to use the helm chart:
 
-- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported)
+- [Helm 3](https://helm.sh/docs/intro/install/) (Helm versions lower than 3.6.0 are not supported)
```

Comment:
What is the origin of this restriction? https://github.com/signalfx/splunk-otel-collector-chart/issues/973 sounds like a bug to me and not necessarily something we want to enforce as a floor.

Reply:
I added it because of this issue. So far I can confirm that trying to use helm version below 3.6.0 causes an "unclosed action" error. I will try to investigate what causes this error

Reply:
Thanks for opening up this PR and bring attention to this. There are a few other things to consider when making a change like this. We opened up https://github.com/signalfx/splunk-otel-collector-chart/pull/979 to help address this. Closing this PR in favor of 979.

Reply:
Makes sense! Thanks :)